src/NzbDrone.Common/Http/HttpClient.cs

```diff
@@ -136,7 +136,16 @@ private HttpResponse ExecuteRequest(HttpRequest request, CookieContainer cookieC
 
             if (request.LogResponseContent)
             {
-                _logger.Trace("Response content ({0} bytes): {1}", response.ResponseData.Length, response.Content);
+                const int MaxContentLength = 50_000;
```

Comment:
Need some advice as to what the max length should be. (50, 100?).

I found 700~ was enough to hang the thread.